### DISCUSSION TITLE
Usability, table 1

### DISCUSSION THREAD

**Discussion by joshichaitanya3** [cssi-pi-community/2022-meeting#109] [Special topics breakouts (Tuesday afternoon)]
This is the start of the Usability breakout room discussion at table 1.

**Comment by joshichaitanya3** [comment]
Why do people want usability?

- MD simulations
- Compiler infrastructure / portals for physics data / computational applications
- Large user base for a data-framework to help users wrangle data / abstracting out preprocessing, etc. *Ease of use* for end-users.
- large hpc system serving science users and developers, accessibility is very important
- Polymer nanocomposites / materials data resource: materials community can have easy access without knowing ontology
- Experimentalists using software written by theory collaborators
- First steps towards building the independent user base
- CFD: Access large datasets from simulations : Virtual sensor idea: specify what part of the data is needed...
- Computer vision / AI
- Mid level API for Science Gateways
- Building a data resource on a Science Gateway : Lower the barrier for entry, easy access.
- Sense making : data tends to be opaque -- how can non CS people get to data / hypothesize, analyze, etc... reduce hurdles.

**Comment by joshichaitanya3** [comment]
Q: How do we measure usability?

**Reply by carolxsong** [reply]
one idea is the time taken to access

who the audiences are

what functions does the user want
utility vs ease of access

what we mean by access

**Comment by joshichaitanya3** [comment]
Q: Literature for UX design?

**Comment by carolxsong** [comment]
Help in user experience/user centered design (books available - reference TBA)

SGCI has consultant to help projects by reviewing and advising on UX (user experience)

**Comment by joshichaitanya3** [comment]
Q: How do we make the user interface more intuitive?

**Comment by joshichaitanya3** [comment]
Q: How do tools to improve usability change with the size of the user base? How to cater to both casual as well as power users?

**Comment by carolxsong** [comment]
example of NMR (or electronic microscopy) needing specialists - research software engineering (RSE) expertise needed but this need not recognized by institutions

**Comment by joshichaitanya3** [comment]
Code bashes (hangout sessions where people try to intentionally break the code to identify bugs) can also be geared towards identifying UX shortcomings.
